calculator/calculator.py

```diff
@@ -0,0 +1,25 @@
+def add(a, b):
+    return int(a) + int(b)
+def subtract(a, b):
+    return int(a) - int(b)
+def multiply(a, b):
+    return int(a) * int(b)
+def divide(a, b):
+    return int(a) / int(b)
+
+print("Select Operation:\n1- Add,\n2- Subtract,\n3- Multiply,\n4- Divide.")
+option = int(input("Enter you choice from 1/2/3/4 :- "))
+num1 = input("Enter your first number :- ")
+num2 = input("Enter your second number :- ")
+if option == 1:
+        print("results : ", add(num1,num2))
```

Comment:
Let's put this logic into the higher-order function which uses previously defined function internally to calculate the results and return them. My personal preference would be to not make functions for executing one-line statements that would be an overkill. e.g.
This would make space for function calling in ram too.
```
def add(a, b):
    return a+b

result =add(a, b)
```
While the following just multiply the a and b
```
result = a+b
```